src/cbmc_viewer/etc/cbmc-viewer-bash-completion.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
Random place for comment: I'd suggest to put this file in `etc/bash_completion.d/` so that it will be found automatically on most systems.

Reply:
Moved to etc/bash_completion.d/cbmc-viewer.sh, but as discussed this can be installed under brew but there is no os-independent way to install under pip.